app/javascript/article-form/components/Help/index.jsx

```diff
@@ -0,0 +1,94 @@
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { ArticleFormTitle } from './ArticleFormTitle';
+import { TagInput } from './TagInput';
+import { BasicEditor } from './BasicEditor';
+import { EditorFormattingHelp } from './EditorFormattingHelp';
+import { Modal } from '@crayons';
+
+const renderModal = (onClose, title, selector) => {
+  const helpHtml =
+    document.getElementById(selector) &&
+    document.getElementById(selector).innerHTML;
+
+  return (
+    <Modal onClose={onClose} title={title}>
+      <div
+        // eslint-disable-next-line react/no-danger
+        dangerouslySetInnerHTML={{ __html: helpHtml }}
+      />
+    </Modal>
+  );
+};
+
+export const Help = ({ previewShowing, helpFor, helpPosition, version }) => {
+  const [state, setState] = useState({
```

Comment:
suggestion (not-blocking) - consider giving this a more specific name than `state` - maybe something like `helpSectionVisibilityState` (maybe you can think of something better than this though!)